x-pack/test/visual_regression/config.js

```diff
@@ -13,7 +13,6 @@ export default async function ({ readConfigFile }) {
     ...functionalConfig.getAll(),
 
     testFiles: [
-      require.resolve('./tests/login_page'),
```

Comment:
Thanks @spalger, actually the login_page is skipped but it still returned a failure for the reason below (also see comment from 2 days ago on this PR), that is why I removed it instead of skipping it like infra suite. 

This is something that recently broke it when I did a merge upstream from master and I don't know why, if you can help me resolve I can add it back and skip it or just remove it totally as it was just a example file. 

With login_page skipped I still got this failure: 

18:09:22 WebDriverError: : Failed to read the 'sessionStorage' property from 'Window': Storage is disabled inside 'data:' URLs.

Reply:
I think I need to move the skip to the describe at Security level maybe then it won't fail, although the failure is new, but I will try that anyway.

describe('Security', () => {
    describe.skip('Login Page', () => {

Reply:
That fixed it.